WordPress/Classes/Utility/Spotlight/Spotlightable.swift

```diff
@@ -9,6 +9,13 @@ class SpotlightableButton: UIButton, Spotlightable {
     var spotlight: QuickStartSpotlightView?
     var originalTitle: String?
 
+    /// The default offset is (-10, 0).
```

Comment:
Nitpicky, but I'd avoid documenting these values here as they may change over time.  I think the comment below is enough to document what this does.